docs/source/index.rst

```diff
@@ -2,12 +2,48 @@
 retworkx Documentation
 ######################
 
+retworkx is a Python package for working with graphs and complex networks. It
+enables the creation, interaction with, and study of graphs and networks.
+
+It provides:
+
+ * Data structures for creating graphs including directed graphs and multigraphs
+ * A library of standard graph algorithms
+ * Generators for various types of graphs including random graphs
+ * Visualization functions for graphs
+
+It is licensed under the
+`Apache 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`__ license and the
+source code is hosted on Github at:
+
+https://github.com/Qiskit/retworkx
+
+retworkx is written in the
+`Rust programming language <https://www.rust-lang.org/>`__ to leverage Rust's
+inherent performance and safety. While this provides numerous advantages
+including significantly improved performance it does mean that the library
+needs to be compiled when being installed from source (as opposed to a pure
+Python library which can just be installed). retworkx supports and publishes
+pre-compiled binaries for Linux on x86, x86_64, aarch64, s390x, and ppc64le,
+MacOS on x86_64, and arm64, and Windows 32bit and 64bit systems. However if
+you're running outside of these platforms to install retworkx you will need a
+rust compiler installed.
```

Comment:
-> "However, if you are not running on one of these platforms, you will need a rust compiler to install retworkx"

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/339/commits/03ad77b9ef846f4d95ae7d059ca927cb299fbc0a